c31-pryhaven/nutshell2
# My event del

Description
Added a delete button for each event. The user can now remove events that have passed.

Applies to issue # (issue number)
#3

Type of change
Confirm that the delete button works by deleting any event.

Please delete options that are not relevant.

  Bug fix (non-breaking change which fixes an issue)
  New feature (non-breaking change which adds functionality)
  Breaking change (fix or feature that would cause existing functionality to not work as expected)
  This change requires a documentation update
Testing Instructions
run git fetch --all in terminal
pull down my-event-del from github.
take data from the example database and plug it into nutshell.json.
run the command json-server -p 8088 -w nutshell.json at ./nutshell2/api
go to root path and run npm start
go to browser and click on events on the nav bar.
click on the delete button for any event. when button is clicked it should remove the event.

Checklist:
  My code follows the style guidelines of this project
  I have performed a self-review of my own code
  I have commented my code, particularly in hard-to-understand areas
  My changes generate no new warnings or errors
  I have added test instructions that prove my fix is effective or that my feature works
